### DISCUSSION TITLE
Detect changes in the background

### DISCUSSION THREAD

**Discussion by Kirukirunen** [ultralytics/ultralytics#13306] [General]
It would be possible to train with images of different backgrounds (the same scene in different states) and segment the appearance of an object that appears in the scene. It would be detecting what you have not been trained to do. You do a great job, great project.

Thank you so much

**Comment by glenn-jocher** [comment]
@Kirukirunen thank you for your kind words and interesting question!

Yes, you can train a model to detect changes in a scene by using images of the same scene under different conditions as part of your training dataset. This approach, often referred to as change detection, involves training the model to recognize the baseline state of the scene and identify deviations from this state.

For segmentation tasks, you might consider using our YOLOv8 models with segmentation capabilities, which can be trained to segment and identify new objects appearing in a scene. Training with a diverse dataset that includes various background states will help the model learn to detect these changes effectively.

If you need further guidance on setting up your training data or configuring the model, feel free to ask. Happy modeling! 🚀

**Reply by Kirukirunen** [reply]
Any recommendations for labeling? The entire background is an object and will the predict (result) return the unknown object? Thank you very much again.

**Reply by glenn-jocher** [reply]
Hello @Kirukirunen,

For labeling, it's essential to ensure that your background is consistently annotated across all images if it's considered an object. You can label the background as a separate class if it plays a crucial role in your model's training.

Regarding the prediction of unknown objects, YOLO models are typically trained to recognize only the classes they've seen during training. If an object doesn't belong to any trained class, it won't be specifically identified as 'unknown'—it simply won't be recognized. However, you can train your model to identify a class as 'unknown' by including diverse objects under this label during training.

Thank you for your continued interest! 🌟
